lucien-ios/Dashboard/LendingCollectionViewCell.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  LendingCollectionViewCell.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/10/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+class LendingCollectionViewCell: UICollectionViewCell {
+    @IBOutlet weak var lendedComicTitle: UILabel!
```

Comment:
Labels should have the word label at the end of their names. `lendedComicTitle` looks like it should just be a String or other data type

Reply:
This should be private